source/draw/gpu/GpuImage.ooc

```diff
@@ -34,7 +34,7 @@ GpuImage: abstract class extends Image {
 	init: func (size: IntVector2D, =_context) { super(size) }
 	resizeTo: override func (size: IntVector2D) -> This {
 		result := this create(size) as This
-		result canvas draw(this, size)
+		DrawState new(result) setInputImage(this) draw()
```

Comment:
The old destination size covered the whole target because `size` equals `result size` from construction.